features/deleteinteractive.feature

```diff
@@ -5,13 +5,36 @@ Feature: Interactives API (Delete interactive)
         When I DELETE "/v1/interactives/0d77a889-abb2-4432-ad22-9c23cf7ee796"
         Then the HTTP status code should be "404"
 
+    Scenario: Cannot delete published interactive
+        Given I am an interactives user
+        And I have these interactives:
+                """
+                [
+                    {
+                        "active": true,
+                        "published": true,
```

Comment:
Using a status field is the approach generally taken in the ONS publishing team.
I think having an active field is ok, as we would want to track the state of an interactive, in case it is reactivated (through a manual db action) we would want it to go back in to the state it was in.

Reply:
we also left 'published' decoupled from state (https://github.com/ONSdigital/dp-interactives-api/blob/ebcf17b96c6d69b6183607a67325111ee0bbb71a/models/interactives.go#L13) because you can still edit after publishing - so we could be in a situation where the state is not really correct - someone edits after published, the import errors but it should still be 'published' with old files - so ImportError but published==true. The UI will reflect the new state and error but the live url will still point to old files until resolved.

we could introduce a new status field maybe - with just `UNPUBLISHED`, `PUBLISHED`? but its just a flag - we could also model the response from our bool.

Reply:
have raised ticket 139 for now.. lets have a discussion and implement whats best